commander/commander_test.go

```diff
@@ -231,14 +231,30 @@ func (suite *CommanderTestSuite) Test_Down_ReturnError_InCaseOfMigratorError() {
 	suite.EqualError(err, "migrating down failed: failure")
 }
 
-func (suite *CommanderTestSuite) Test_Down_ReturnNil_InCaseOfSuccess() {
+func (suite *CommanderTestSuite) Test_Down_ReturnError_InCaseOfMissingArgumentN() {
 	// Arrange
 	suite.flagSet.String("path", "", "")
 	suite.flagSet.String("url", "", "")
 	if err := suite.flagSet.Parse([]string{"--path", "testdata", "--url", "connectionurl"}); err != nil {
 		suite.FailNow(err.Error())
 	}
-	suite.migratorMock.On("Migrate", "testdata", "connectionurl", direction.Down, 0, 1).Return(nil).Once()
+
+	// Act
+	err := suite.commander.Down(suite.ctx)
+
+	// Assert
+	suite.Error(err)
```

Comment:
This may be removed due to the next line including it.

Reply:
I fixed all the error checking issues in another branch :) I think